src/test/script_tests.cpp

```diff
@@ -787,9 +787,10 @@ BOOST_AUTO_TEST_CASE(script_PushData)
 }
 
 CScript
-sign_multisig(CScript scriptPubKey, std::vector<CKey> keys, CTransaction transaction)
+sign_multisig(const CScript& scriptPubKey, std::vector<CKey> keys, const CTransaction& transaction, CAmount amt)
```

Comment:
I think we need fix also the qa test `rawtransactions.py` and the RPC `signrawtransaction`  to take into account of the mandatory presence of a non zero value for amount.